content/education/manage-coursework-with-github-classroom/manage-classrooms.md

```diff
@@ -58,8 +58,10 @@ If your course already has a roster, you can update the students on the roster o
 {% data reusables.classroom.click-students %}
 1. To connect {% data variables.product.prodname_classroom %} to your LMS and import a roster, click {% octicon "mortar-board" aria-label="The mortar board icon" %} **Import from a learning management system** and follow the instructions. For more information, see "[Connect a learning management system to {% data variables.product.prodname_classroom %}](/education/manage-coursework-with-github-classroom/connect-a-learning-management-system-to-github-classroom)."
     !["Import from a learning management system" button](/assets/images/help/classroom/click-import-from-a-learning-management-system-button.png)
-1. To create a roster manually, type your student identifiers. Optionally, click **Upload a CSV or text file** to upload a file containing the identifiers.
-    ![Text field for typing student identifiers and "Upload a CSV or text file" button](/assets/images/help/classroom/type-or-upload-student-identifiers.png)
+1. Choose whether to import a roster or create a roster manually.
```

Comment:
For a step like this, with two-sub steps, we want the original step to be a summary of all sub-steps. Choosing whether you want to import a roster or creating one manually is really a different step than doing the actual importing. However, "create your roster" is a good summary, and the user can choose to accomplish the step either by importing or typing. 😊 

```suggestion
1. Create your class roster.
```